go-controller/pkg/crd/adminpolicybasedroute/v1/types.go

```diff
@@ -58,6 +58,8 @@ type ExternalNetworkSource struct {
 // ExternalNextHops contains slices of StaticHops and DynamicHops structures. Minimum is one StaticHop or one DynamicHop.
 type ExternalNextHops struct {
 	// StaticHops defines a slice of StaticHop. This field is optional.
+	// Defining a soft maximum items to 8192 to enable XValidations for the IP field. It is possible to increase this value in the future if there is a need to support more than 8192 static gateway IPs in a single cluster.
```

Comment:
Explaining why you are making this change should be preceded with a `+` sign so that it does not end up in the API documentation. You can change the line above to something like `// StaticHops defines a slice of up to 8192 static hops`